lib/ReactViews/Map/Panels/SharePanel/SharePanel.jsx

```diff
@@ -143,39 +148,55 @@ const SharePanel = createReactClass({
                        onOpenChanged={this.onOpenChanged}
                        smallScreen={this.props.viewState.useSmallScreenInterface}>
                 <If condition={this.state.isOpen}>
+                        <If condition={this.props.viewState.useSmallScreenInterface}>
```

Comment:
Probably would be better to break into renderSmallScreen and renderNormal functions because the layouts are so different